### PR TITLE
✨ Add send email notification & delayed redirecting to login page

### LINKED ISSUE #47: Forgot Password Page - Send email notification
### To Do
- [ ] After the user has entered and confirmed his email, a message should be displayed that an email has been sent to reset his password.

### PR DESCRIPTION
**Pull request by Dev-dfm**
Forgot password page:
- Add send email notification as conditional rendering
- Add redirecting to login page after a delay of 6s

Try it out:
https://fit-2-gether-dev-dfm-is-chpow7.herokuapp.com/

fixes #47

### CODE REVIEW

File: src/app/pages/ForgotPassword/ForgotPassword.tsx [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
-export default function ForgotPassword(): JSX.Element {
+function ForgotPassword(): JSX.Element {
+  const history = useHistory();
   const [email, setEmail] = useState('');
+  const [isSend, setIsSend] = useState(false);
+
+  async function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+
+    setIsSend(!isSend);
```
Comment (lmachens) [inline src/app/pages/ForgotPassword/ForgotPassword.tsx:17]: It feels weird if you submit the form two times in a row (click fast).

File: src/app/pages/ForgotPassword/ForgotPassword.tsx [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
 import LabeledInput from '../../components/LabeledInput/LabeledInput';
 import styles from './ForgotPassword.module.css';
 
-export default function ForgotPassword(): JSX.Element {
+function ForgotPassword(): JSX.Element {
+  const history = useHistory();
   const [email, setEmail] = useState('');
+  const [isSend, setIsSend] = useState(false);
+
+  async function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
```
Comment (lmachens) [inline src/app/pages/ForgotPassword/ForgotPassword.tsx:14]:
```suggestion
  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
```

File: src/app/pages/ForgotPassword/ForgotPassword.tsx [hunk 3]
```diff
@@ -22,25 +35,34 @@ export default function ForgotPassword(): JSX.Element {
...
+              label="email"
+              type="email"
+              value={email}
+              onChange={setEmail}
+              required
+            />
+          </div>
+          {isSend && (
+            <p className={styles.passwordForm__Notification}>
+              {' '}
```
Comment (lmachens) [inline src/app/pages/ForgotPassword/ForgotPassword.tsx:56]:
```suggestion
```

File: src/app/pages/ForgotPassword/ForgotPassword.tsx [hunk 4]
```diff
@@ -22,25 +35,34 @@ export default function ForgotPassword(): JSX.Element {
...
+              Your password was reseted. Please check your Email. You will be
+              redirected to the login page.{' '}
```
Comment (lmachens) [inline src/app/pages/ForgotPassword/ForgotPassword.tsx:58]:
```suggestion
              redirected to the login page.
```

File: src/app/pages/ForgotPassword/ForgotPassword.tsx [hunk 5]
```diff
@@ -22,25 +35,34 @@ export default function ForgotPassword(): JSX.Element {
...
-            label="email"
-            type="email"
-            value={email}
-            onChange={setEmail}
-            required
-          />
-        </form>
-        <div className={styles.resetButton}>
+        <form onSubmit={handleSubmit} className={styles.passwordForm_form}>
+          <div>
```
Comment (lmachens) [inline src/app/pages/ForgotPassword/ForgotPassword.tsx:43]: I don't think you need this `div`

File: src/app/pages/Login/Login.tsx [hunk 6]
```diff
@@ -67,9 +67,7 @@ function Login(): JSX.Element {
           <Button variant="secondary">Log in</Button>
 
           {errorMessage && (
-            <div className={styles.registerForm__error}>
-              Error: {errorMessage}
-            </div>
+            <div className={styles.registerForm__error}>{errorMessage}</div>
```
Comment (lmachens) [inline src/app/pages/Login/Login.tsx:70]: Not related to this change, but I noticed that the class is called `registerForm`, but you are in the `Login` page.

**Review by lmachens — COMMENTED** [review]
gj, but I would mention that it is doesn't really reset the password / the backend functionality is missing.

**Review by Niklas-Schaefers — APPROVED** [review]
lgtm 🚀

**Review by dhnazarenko — COMMENTED** [review]
LGTM 🚀

**Pull request merged**